serrano/resources/preview.py

```diff
@@ -133,21 +151,15 @@ def get(self, request):
         model_name = opts.verbose_name.format()
         model_name_plural = opts.verbose_name_plural.format()
 
-        data = self.get_page_response(request, paginator, page)
-
-        data.update({
+        data = {
             'keys': header,
             'items': objects,
             'item_name': model_name,
             'item_name_plural': model_name_plural,
-            'item_count': paginator.count
-        })
-
-        path = reverse('serrano:data:preview')
-        links = self.get_page_links(request, path, page, extra=params)
```

Comment:
Should we make an attempt at returning page links? If given a `page`, we could still probably generate a `previous` and `next` link but there is no guarantee that the `next` link will have any results so I'm not sure of the utility. The client might need to sort this out for themselves...